### ISSUE TITLE
Training & val loss

### ISSUE THREAD

**Issue by agunapal** [wangtianrui/DCCRN#6]
Hello,
Could you please share how many epochs did you train the model for. 

I see that the training loss plateaus around -21 and then remains around there for many epochs.

Also, how do i interpret the loss value? and how do you know when the training is done.

**Comment by wangtianrui** [comment]
The code has an automatic learning rate adjustment mechanism and early stop mechanism.
Adjustment mechanism: The learning rate is reduced by 50 percent if the loss of 3 epochs in the test set is not decreased
Early stop mechanism: The loss of 20 epochs in the test set does not drop, and the training stops. And keep the best model
```python
# BaseModel.py
if self.conf["train"]["half_lr"]:
    self.scheduler = ReduceLROnPlateau(
        optimizer=self.optimizer, factor=self.conf["scheduler"]["factor"],
        patience=self.conf["scheduler"]["patience"], verbose=self.conf["scheduler"]["verbose"]
    )
if self.conf["train"]["early_stop"]:
    self.early_stop = EarlyStopping(monitor="val_loss", patience=20, verbose=True)
````
You can change the corresponding values according to your needs. In my memory, training set losses of -23.5 or more are better

**Comment by agunapal** [comment]
Thank you. I guess there is some issue with the prediction code? .Even after training for 1 epoch, when I run inference, I get the audio output to be the same as input. I would have expected the output to be worse than the input

**Comment by wangtianrui** [comment]
Could you please show your code and results? Is the spectrogram the same?

**Comment by agunapal** [comment]
Thanks for getting back. I just needed to train more. I am seeing a difference in the spectrogram. Will continue training 
![inp](https://user-images.githubusercontent.com/16617092/125864315-7a2b84a1-42ac-4aa7-b54f-f2ace50ecbba.png)
![out](https://user-images.githubusercontent.com/16617092/125864336-c14bcfcd-3c57-4841-99c1-f76e75c2b67b.png)

**Comment by hdmjdp** [comment]
@agunapal can you show me the loss log, why my loss is around 12 in epoch 2?
